app/assets/stylesheets/cypress/_buttons.scss

```diff
@@ -22,8 +23,9 @@ a.cmd[disabled], a.cmd[disabled]:hover {
   cursor: not-allowed;
 }
 
+// button hover color
 a.cmd:hover, input.cmd:hover {
-  background-color: $btnHoverBkg;
+  background-color: #ccc; // gray
```

Comment:
This is the only place `$btnHoverBkg` is used (as far as I can tell), so we may just want to change the value of the variable?